### ISSUE TITLE
Very slow server. Changed the IP address 

### ISSUE THREAD

**Issue by Esteidinger** [Nyr/openvpn-install#422]
I setup an OpenVPN server using this tutorial recently for the first time. I have used OpenVPN with other tutorials in the past on similar hardware with fantastic results. (ubuntu 14.04 in AWS on a free tier T2.micro this time I used 16.04 and my old tutorial did not work so I found this one.) 

When selecting the IP address I poorly chose to change the value that the system picked up. 
Now the OpenVPN server runs incredibly slowly as compared to my old server (which happens to still be running)  Can I change this config value?  

It doesn't seem to exist in server.conf

**Comment by Nyr** [comment]
You can run the script again and it will offer the option to uninstall, then run it one more time and you are able to start a new setup. But I think that you are a bit confused: if the IP address is wrong, the setup will simply not work. It will not just be a bit slower.

If I had to take a guess, I think that the micro instance is just slow. The script also uses strong crypto, which could be a bit slower than what you were previously using.

**TL;DR:** get a faster server, or one which is closer to your location.
